app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -1,9 +1,7 @@
 module Idv
   module DocumentCaptureConcern
     def override_document_capture_step_csp
-      if !IdentityConfig.store.doc_auth_document_capture_controller_enabled
```

Comment:
Since this condition was only taking effect when the feature was disabled, should it be removed altogether?

Reply:
Problem here is that the DocumentCapture step is still used in the hybrid flow, so the flow state machine still needs it. Hm... I wonder if `doc_auth_controller` could stop including this since we removed DocumentCapture from this flow. `capture_doc_controller` still needs it.